src/Cache/Cache.php

```diff
@@ -168,7 +168,18 @@ protected static function _buildEngine($name)
         }
 
         $config = static::$_config[$name];
-        $registry->load($name, $config);
+
+        try {
+            $registry->load($name, $config);
+        } catch (RuntimeException $e) {
+            if (!array_key_exists('fallback', $config)) {
```

Comment:
That means https://github.com/cakephp/cache would require `cakephp/log`, yeah? I'm not sure how the splits are managed, so not sure what I would need to do to allow it.